src/components/DataTable/DataTable.spec.jsx

```diff
@@ -491,6 +492,51 @@ describe('DataTable', () => {
 				assert.equal(thFirstRowArray[3].text(), 'Occupation', 'must render correct title');
 				assert.equal(thFirstRowArray[3].prop('rowSpan'), 2, 'rowSpan must be 2');
 			});
+
+			it('should default to align=center', () => {
+				const wrapper = shallow(
+					<DataTable data={testData}>
+						<Column field='id' title='ID'/>
+						<ColumnGroup title='Name'>
+							<Column field='first_name' title='First'/>
+							<Column field='last_name' title='Last'/>
+						</ColumnGroup>
+						<Column field='email' title='Email'/>
+						<Column field='occupation' title='Occupation'/>
+					</DataTable>
+				);
+
+				const thWrapper = wrapper
+					.find(ScrollTable).shallow()
+					.find(ScrollTable.Thead).shallow()
+					.find(ScrollTable.Tr).first().shallow()
+					.find(ScrollTable.Th).at(1).shallow()
+
+				assert(thWrapper.hasClass('lucid-Table-align-center'), 'must be true');
```

Comment:
the useful message should be `must have correct className` or something like that.